src/ucs/memory/numa.c

```diff
@@ -10,10 +10,34 @@
 
 #include "numa.h"
 
+#include <ucs/datastruct/khash.h>
 #include <ucs/debug/assert.h>
 #include <ucs/debug/log.h>
+#include <ucs/sys/string.h>
+#include <ucs/sys/sys.h>
+#include <ucs/type/spinlock.h>
 #include <stdint.h>
 #include <sched.h>
+#include <dirent.h>
+
+#define UCS_NUMA_DEFAULT_NODE            0
+#define UCS_NUMA_DEFAULT_DISTANCE        UCS_NUMA_MIN_DISTANCE
+#define UCS_NUMA_MAX_NODE                UINT16_MAX
+#define UCS_NUMA_CORES_DIR_PATH          "/sys/devices/system/cpu"
```

Comment:
WDYT of removing duplication of `/sys/devices/system` here twice,
and also in `cpu.c`?